### PR TITLE
Add mapzen isolines

### PR DESCRIPTION
**Pull request by iriberri**

### CODE REVIEW

**Comment by rafatower** [comment]
:+1: to a046d3ce977cec08b7106fc266325f0c99d2700f

File: server/lib/python/cartodb_services/cartodb_services/mapzen/isolines.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
+from math import cos, sin, tan, sqrt, pi, radians, degrees, asin, atan2
+import logging
```
Comment (iriberri) [inline server/lib/python/cartodb_services/cartodb_services/mapzen/isolines.py]: remove/comment this for prod

File: server/extension/sql/80_isolines_helper.sql [hunk 2]
```diff
@@ -53,3 +53,74 @@ RETURNS SETOF cdb_dataservices_server.isoline AS $$
...
+    quota_service.increment_success_service_use()
+    quota_service.increment_isolines_service_use(len(isolines))
+    return result
+  except BaseException as e:
+    import sys, traceback
+    type_, value_, traceback_ = sys.exc_info()
+    quota_service.increment_failed_service_use()
+    error_msg = 'There was an error trying to obtain isolines using mapzen: {0}'.format(e)
+    plpy.debug(traceback.format_tb(traceback_))
+    raise e
```
Comment (iriberri) [inline server/extension/sql/80_isolines_helper.sql]: use plpy.error here

File: server/lib/python/cartodb_services/cartodb_services/mapzen/isolines.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+    Returns:
+        Array of {lon: x, lat: y} as a representation of the isoline
+    """
+    def calculate_isoline(self, origin, costing_model, isorange, upper_rmax, cost_variable, unit_factor=1.0):
+
+        # NOTE: not for production
+        #logging.basicConfig(level=logging.DEBUG, filename='/tmp/isolines.log')
+        #logging.basicConfig(level=logging.DEBUG)
+        logging.debug('origin = %s' % origin)
```
Comment (iriberri) [inline server/lib/python/cartodb_services/cartodb_services/mapzen/isolines.py]: get rid of this

File: server/lib/python/cartodb_services/cartodb_services/mapzen/isolines.py [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+            response = self._matrix_client.one_to_many([origin] + location_estimates,  costing_model)
+            costs = [None] * self.NUMBER_OF_ANGLES
+            for idx, c in enumerate(response['one_to_many'][0][1:]):
+                if c[cost_variable]:
+                    costs[idx] = c[cost_variable]*unit_factor
+                else:
+                    costs[idx] = isorange
+
+            logging.debug('i = %d, costs = %s' % (i, costs))
```
Comment (iriberri) [inline server/lib/python/cartodb_services/cartodb_services/mapzen/isolines.py]: remove this

**Pull request merged**